Doc/reference/compound_stmts.rst

```diff
@@ -302,8 +302,7 @@ stored in the :mod:`sys` module and can be accessed via :func:`sys.exc_info`.
 :func:`sys.exc_info` returns a 3-tuple consisting of the exception class, the
 exception instance and a traceback object (see section :ref:`types`) identifying
 the point in the program where the exception occurred.  :func:`sys.exc_info`
-values are restored to their previous values (before the call) when returning
-from a function that handled an exception.
+elements are restored to their previous values when leaving an exception handler.
```

Comment:
Actually, it would probably be better to write ":func:`sys.exc_info`is restored to its previous value..."      speaking of its elements implies that the tuple is the same but its elements changed their values.

Reply:
Except that sys.exc_info is a function, so that wouldn't be right either (the value of exc_info does not change). Maybe it's better how it was - ":func:`sys.exc_info` values are restored to their previous values" .      

"sys.exc_info values" here refers to the values returned by sys.exc_info.

Reply:
@iritkatriel After reading again the whole paragraph I added a more explicit description and a code example.

Reply:
Thanks for your suggestions, I think the doc is more correct and readable now.